### DISCUSSION TITLE
Checkliste Release

### DISCUSSION THREAD

**Discussion by SebStaeubert** [medizininformatik-initiative/kerndatensatzmodul-consent#62] [Show and tell]
Siehe https://github.com/medizininformatik-initiative/kerndatensatz-meta/wiki/Release-Check-List bzw. https://github.com/medizininformatik-initiative/kerndatensatz-meta/wiki/Module-Release-Workflow

- [ ] Github - Dateien committen
  - [ ] ressourcen-profile
  - [ ] searchparameters
  - [ ] terminologie
  - [ ] Pull Request develop -> main
  - [ ] Release mit der Version als Tag und Changelog erstellen
- [ ] Simplifier
  - [ ] Github-Sync
  - [ ] IG
    - [ ] IG Settings anpassen
      - [ ] Titel mit Versions-Jahr: "MII IG Consent v2025"
      - [ ] Version: "2025.0.0"
      - [ ] Visibility: Public
      - [ ] URL Key mit Versions-Jahr: "mii-ig-modul-consent-2025"
      - [ ] Scope mit Versions-Jahr setzen "de.medizininformatikinitiative.kerndatensatz.consent 2025.0.0"
      - [ ] Achtung bei (nachträglichen) Änderungen:
        - [ ] URL Key mit dem zu ändernden Versions-Jahr: "mii-ig-modul-consent-2025"
        - [ ] Scope mit dem zu ändernden Versions-Jahr setzen "de.medizininformatikinitiative.kerndatensatz.consent 2025.0.0", damit diese Version geändert wird!
        Hinweis: die URL-Keys und Scopes erzeugen jeweils Instanzen des IG, die jeweils Versionen haben. Editiert werden kann aber in der Simplifier IG Seite des Projektes immer nur eine Instanz. Durch setzen des URL-Key und Scope kann aber eine neue Version einer Instanz erzeugt werden. Wird ein bisher nicht verwendeter URL-Key genutzt, wird eine neue Instanz des IG erzeugt.
    - [ ] IG aktualisieren Inhalt (alle Seiten)
    - [ ] IG Page Release Notes um die [Release Notes der aktuellen Version](https://github.com/medizininformatik-initiative/kerndatensatzmodul-consent/releases) (github) ergänzen
    - [ ] IG Version aktualisieren ([index.guide.md](https://simplifier.net/editguide/medizininformatikinitiative-modulconsent-implementationguide/editor?filepath=IGMIIKDSModulConsent%2Findex.guide.md))
    - [ ] IG exportieren (export IG nutzen; publish ist nicht erforderlich)
  - [ ] [Package Ressource](https://simplifier.net/MedizininformatikInitiative-ModulConsent/package/~overview) (Manifest) anpassen
  - [ ] [Package erzeugen](https://simplifier.net/MedizininformatikInitiative-ModulConsent/~releases) (Package Description anpassen, Release notes angeben)
- [ ] QS:
  - [ ] HAPI FHIR Server (Docker) hochfahren und Package hochladen
  - [ ] ggf. Fehlermeldungen beheben
- [ ] Mail an MII Koordinierungsstelle mit zur Veröffentlichung auf der [MII Homepage](https://www.medizininformatik-initiative.de/Kerndatensatz/KDS_Consent_V2025/MII-IG-Modul-Consent.html) (ggf. [Link](https://www.medizininformatik-initiative.de/de/basismodule-des-kerndatensatzes-der-mii) anpassen)
- [ ] [KDS-Meta Modul-Übersicht](https://github.com/medizininformatik-initiative/kerndatensatz-meta/wiki/%C3%9Cbersicht-%C3%BCber-Versionen-der-Kerndatensatz%E2%80%90Module) aktualisieren
- [ ] Infomail an MII Koordinierungsstelle (Links anpassen):

> wir haben das Release 2025.0.x des KDS Modul Consent erstellt.
> 
> Github: https://github.com/medizininformatik-initiative/kerndatensatzmodul-consent/releases/tag/2025.0.0
> Package-Download: https://simplifier.net/packages/de.medizininformatikinitiative.kerndatensatz.consent/2025.0.0
> IG (Simplifier): https://simplifier.net/MedizininformatikInitiative-ModulConsent/~guides
> IG (Preview): https://simplifier.net/guide/medizininformatikinitiative-modulconsent-implementationguide?version=current
> IG (Download): https://simplifier.net/guide/medizininformatikinitiative-modulconsent-implementationguide/$exportaszipui
> 
> Ein Change-Log steht in der Beschreibung des Releases (github) und des Packages.

**Comment by oeme-github** [comment]
Cool und herzlichen Glückwunsch zum Release :-)
